lib/src/widgets/yaru_info.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:yaru/yaru.dart';
+
+enum YaruInfoType {
+  note,
```

Comment:
Isn't `info` more used in this context?

Reply:
I thought the same... but YaruInfoType.info reads weird :)
I got obv inspired by the github badges

> [!NOTE]  
> Highlights information that users should take into account, even when skimming.

> [!TIP]
> Optional information to help a user be more successful.

> [!IMPORTANT]  
> Crucial information necessary for users to succeed.

> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.

> [!CAUTION]
> Negative potential consequences of an action.